src/data/STATUSES/WAR.js

```diff
@@ -36,21 +33,38 @@ export default {
 		icon: 'https://xivapi.com/i/012000/012551.png',
 	},
 
+	SHAKE_IT_OFF: {
```

Comment:
Shake is technically a party buff, but then again a bunch of non-offensive similar skills (eg Mantra) are in job-specific statuses so derp.